### ISSUE TITLE
Remove currently published News Articles from the collection.

### ISSUE THREAD

**Issue by HugPhiluu** [PrismLauncher/prismlauncher.github.io#12]

**Comment by Regular-Baf** [comment]
I don't think we should remove them, but possibly flag them as archived or something?

**Comment by jschenke488** [comment]
> I don't think we should remove them, but possibly flag them as archived or something?

I agree with this, probably just gray them out and put an icon next to it saying archived

**Comment by ZekeSmith** [comment]
I am not removing them for now however I will work on an archive system
